tests/PayU/Alu/RequestTest.php

```diff
@@ -15,12 +15,37 @@ class RequestTest extends \PHPUnit_Framework_TestCase
      */
     private $order;
 
+    /**
+     * @var FX
+     */
+    private $fx;
+
+    /**
+     * @var MerchantConfig
+     */
+    private $cfg;
+
+    /**
+     * @var Billing
+     */
+    private $billing;
+
+    /**
+     * @var Delivery
+     */
+    private $delivery;
+
+    /**
+     * @var User
+     */
+    private $user;
+
     public function setUp()
     {
 
-        $cfg = new MerchantConfig('MERCHANT_CODE', 'SECRET_KEY', 'RO');
+        $this->cfg = new MerchantConfig('MERCHANT_CODE', 'SECRET_KEY', 'RO');
```

Comment:
I'm not a fan of keeping unnecessary state. Can't you just pass variables to functions, as before?